Makefile.util.def

```diff
@@ -730,12 +730,6 @@ script = {
   dependencies = 'garbage-gen$(BUILD_EXEEXT)';
 };
 
-script = {
```

Comment:
Can we leave a comment in the gitlab-ci.yml that we disabled those tests and why?
Also, the mknods in the gitlab-ci.yml are likely not needed anymore. They were addedd as an attempt to make those tests pass.